src/components/Actions/Actions.vue

```diff
@@ -82,6 +82,20 @@ https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-action
 ```
 
 ### Custom icon slot
+To be used with `vue-material-design-icons` only. For icon classes use the `default-icon` slot.
+It can be used with one or multiple actions.
+```
+<Actions>
+	<MicrophoneOff
```

Comment:
This looks weird. Should be inside an ActionButton or something?

Reply:
Why does it look weird to you?  And no the actionbutton is something else

Reply:
I missed the `slot` line